de.gonicus.gonnect.yml

```diff
@@ -0,0 +1,96 @@
+id: de.gonicus.gonnect
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: gonnect
+
+finish-args:
+  - --device=all
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pcsc
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.mpris.MediaPlayer2.*
```

Comment:
what is `org.mpris.MediaPlayer2.*` for?

Reply:
This is the standard way to control external players. See https://specifications.freedesktop.org/mpris-spec/latest/ .

Reply:
I know that, but why do you need this? what are you using external players for?

Reply:
Incoming calls are stopping potentially running music / video / whatever.